lib/ansible/modules/windows/win_package.py

```diff
@@ -130,6 +130,13 @@
     type: bool
     default: yes
     version_added: '2.4'
+  log_path:
+    description:
+    - Full path of log file.
```

Comment:
It is nice when the first entry is a description of the what the option does. Maybe?

```
log_path:
  description:
  - Specifies the path to a log file that is persisted after an MSI package is installed or uninstalled.
  - When omitted, a temporary log file is used for MSI packages.
  - This is only valid for MSI files, use C(arguments) for other package types.
```

Feel free to change it how you wish.

Reply:
thanks ... those words look better ... done